src/nvim/normal_cmds.lua

```diff
@@ -0,0 +1,1157 @@
+local bit = require('bit')
+
+local NONE = 0
+local NCH = 0x01
+local NCH_NOP = bit.bor(0x02, NCH)
+local NCH_ALW = bit.bor(0x04, NCH)
+local LANG = 0x08
+local SS = 0x10
```

Comment:
Can we now remove the C definitions of these (and the others...) flags?

Reply:
these flags still used in some functions as condition.